src/ceph-disk/ceph_disk/main.py

```diff
@@ -603,7 +603,8 @@ def block_path(dev):
         return dev
     path = os.path.realpath(dev)
     rdev = os.stat(path).st_rdev
-    (M, m) = (os.major(rdev), os.minor(rdev))
+    M = ((rdev >> 8) & 0xfff) | ((rdev >> 32) & ~0xfff)
```

Comment:
i'd suggest monkey patching `os.major()` and `os.minor()`. in other words, add following code snippet at the beginning of `main.py`, right before `class Ptype(object)`
 
```py
try:
    # see https://bugs.python.org/issue23098
    os.major(0x80002b00)
except OverflowError:
    os.major = lambda devid: ((devid >> 8) & 0xfff) | ((devid >> 32) & ~0xfff)
    os.minor = lambda devid: (devid & 0xff) | ((devid >> 12) & ~0xff)
```

Reply:
@tchaikov , thanks your detailed instruction. ^^
 i've done with above monkey patch, please review again.

Reply:
please drop this change. as it is not necessary anymore.

Reply:
OK. done.

Reply:
no, you reverted the change i suggested.